tests/syntax/test_requirements.py

```diff
@@ -86,6 +86,14 @@ def test_runtime_parse_error_in_requirement():
     with pytest.raises(ScenicSyntaxError):
         sampleScene(scenario)
 
+def test_soft_requirement_with_temporal_operators():
+    "Temporal operators are not be allowed if prob != 1"
```

Comment:
Extra "be"

Reply:
Fixed in f30d11e4dcf3bcc6d8a894fc66b753e9c2af6c2d